spec/examples.txt

```diff
@@ -0,0 +1,8 @@
+example_id                                                | status | run_time        |
```

Comment:
@himdel it's part of master for some reason, I opened a bug https://github.com/ManageIQ/manageiq/issues/7535